index.html

```diff
@@ -1,54 +1,183 @@
+      </ul>
+    </nav>
+    <div class="header-text">
+      <h3>Hi! You want your ideas turned to reality?</h3>
+      <h1>
+        Idreality is all you need. You've just gotten all you need at one
+        stop.
+      </h1>
+      <div class="border-text">
+        <p>
+          Idreality is an innovation and life coaching hub, a digital platform
+          that provides solutions to the ideas that have not been birthed,
+          taking them from incubation to execution. It is a free online space
+          to accommodate the youth and their ideas, and walk them towards
+          realization of their potential.
+        </p>
+      </div>
+      <div class="header-date">
+        <h2>~iDreality! Where the ideas come to realization~</h2>
+        <p>@iHub Kenya, Caren Shopping Centre.</p>
+      </div>
+    </div>
+  </header>
+  <section class="our-program">
+    <div class="program">
+      <div class="program-header">
+        <h2>Who Are iDreality Participants?</h2>
+        <hr />
+      </div>
+      <div class="program-list">
+        <div class="program-list-items">
+          <i class="fa-solid fa-users icon-bar"></i>
+          <h2>Ideator</h2>
+          <p>
+            This user joins the forum and shares daily problems faced at job,
+            home, or anywhere. The user helps the Dorks to innovate faster,
+            smarter, and easier
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-user-doctor icon-bar"></i>
+          <h2>Dork</h2>
+          <p>
+            This user is a techie. Dork connotes an expert or enthusiast
+            obsessed with a hobby in programming. The user eats and breathes
+            code
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-people-carry-box icon-bar"></i>
+          <h2>Aid</h2>
+          <p>
+            These are the users of development assistance to iDreality. They
+            help in the development of iDreality site. They also help in handy
+            with donations
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-user-tie icon-bar"></i>
+          <h2>Financier</h2>
+          <p>
+            This user is a sponsor. The user introduces and supports a
+            proposal for implementation. The user supports the Ideator and the
+            Dork financially to actualize their project.
+          </p>
+        </div>
+        <div class="program-btn">
+          <button type="button">Join A Group</button>
+        </div>
+      </div>
+
+      <div class="program-btn-desktop">
+        <p><a href="#">Join a Group</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="experts">
+    <div class="expert-header">
+      <h2>Featured Contributors</h2>
+      <hr />
+    </div>
+
+    <div class="expert-list"></div>
```

Comment:
[OPTIONAL]

- [ ] It would a great thing if you would take care of the experts'  list. The pictures resize and after jumping from mobile to desktop,  is not looking great.
![image](https://user-images.githubusercontent.com/83596135/157730698-2431a4e1-5649-41de-8ece-a9f8856c2373.png)